app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarViewModel.kt

```diff
@@ -1,5 +1,6 @@
 package com.pvp.app.ui.screen.calendar
 
+import android.util.Log
```

Comment:
Is this used, if only import appeared, but not the actual usage of it?

If not used, should remove it.

Also, there is an option to click onto `app` package and select `Optimize Imports` (before the final commit) to clean unused imports through out all the files under the package. `Control+Alt+O` does the same in current file, if I recall correctly.